apps/web/src/server/views/applications/components/case-form/case/title.njk

```diff
@@ -1,36 +1,36 @@
-{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
 {% macro title(name, values, errors) %}
+	{% if name == 'titleWelsh' and values['title'] != null %}
 
-	{% set labelText = 'Name of the project' %}
-	{% if name == 'titleWelsh'%}
-		{% set labelText = 'Name of the project in Welsh' %}
-	{% endif %}
-
+    {% set labelText = 'Name of the project' %}
```

Comment:
The label "Name of the project" should probably change to "Project name" to keep the english and welsh fields consistent.